tests/test_metadata.py

```diff
@@ -262,6 +271,14 @@ def test_open_access_content_mirrored(self):
         )
         assert book.mirror_url.endswith(expect)
 
+        # make sure the mirrored link is safely on edition
+        assert edition.license_pool.identifier.links[0].resource.representation.mirror_url.startswith('http://s3.amazonaws.com/test.content.bucket/')
```

Comment:
How about:
        for edlink in edition.license_pool.identifier.links:
            edmirror = resource.representation.mirror_url
            assert edmirror.startswith('http://s3.amazonaws.com/test.content.bucket/')